docker-compose.yml

```diff
@@ -1,24 +1,49 @@
 version: '3'
 services:
-  web:
-    build: .
+
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: mysterium-api
     ports:
-      - "5000:5000"
+      - 8001:5000
+    environment:
+      DB_HOST: mysterium-mysql
+      DB_NAME: myst_api
+      DB_USER: myst_api
+      DB_PASSWORD: myst_api
+    volumes:
+      - .:/code
     depends_on:
       - db
 
   dashboard:
     build:
       context: .
       dockerfile: Dockerfile.dashboard
+    container_name: mysterium-dashboard
```

Comment:
`mysterium-api`, `mysterium-dashboard` and `mysterium-dashboard-mysql` are very long and hard to use names - can use `myst` instead of `mysterium`?

Reply:
I tried to relate ti node name `mysterium-node`

Reply:
Maybe `myst-node` is enough as well?

Reply:
?